.isort.cfg

```diff
@@ -0,0 +1,10 @@
+[settings]
```

Comment:
This file is not needed anymore. The isort config is now being defined in the pyproject.TOML!

Reply:
Hmmm, I was getting all kinds of flake errors that were related to the sort order and when I dropped this file in, all was good. I guess that means there are some settings that need to be moved from this file into the pyproject.TOML.

Reply:
Updated pyproject.TOML and removed .isort.cfg.

The problem was that previously, we had been marking grasshopper as `known_local_folder` instead of `known_first_party`.

This is how I had been using the sections.
local folder == this package, in this case grasshopper
first party == other Alteryx packages, in this case none
third party == external packages
stdlib == python3 standard library imports

Let me know if you think we should change how we manage now that grasshopper is open source?